src/modules/Elsa.Elasticsearch/Common/Utils.cs

```diff
@@ -0,0 +1,43 @@
+using Elsa.Elasticsearch.Services;
+
+namespace Elsa.Elasticsearch.Common;
+
+public static class Utils
+{
+    public static string GenerateIndexName(string aliasName)
+    {
+        var now = DateTime.Now;
+        var month = now.ToString("MM");
+        var year = now.Year;
+        var day = now.Day;
+        var hour = now.Hour;
+        var minute = now.Minute;
+
+        return aliasName + "-" + year + "-" + month + "-" + day + hour + minute;
+    }
+
+    public static IEnumerable<Type> GetElasticConfigurationTypes() =>
+        AppDomain.CurrentDomain.GetAssemblies()
+            .SelectMany(s => s.GetTypes())
+            .Where(p => 
+                typeof(IElasticConfiguration).IsAssignableFrom(p) && 
+                p is {IsClass: true, IsAbstract: false});
+
+    public static IEnumerable<Type> GetElasticDocumentTypes() =>
+        GetElasticConfigurationTypes()
+            .Select(t => t.BaseType!.GenericTypeArguments.First()).ToList();
+    
+    public static IDictionary<Type, string> ResolveAliasConfig(
+        IDictionary<Type, string> defaultConfig, 
+        IDictionary<string, string>? option1, 
+        IDictionary<string, string>? option2)
```

Comment:
The names `option1` and `option2` are too generic and too similar - can you rename it to something that makes their purpose more evident?